app/models/draft.rb

```diff
@@ -58,13 +58,15 @@ def self.stream(opts = nil)
         pu.username, pu.name, pu.id user_id, pu.uploaded_avatar_id, pu.username_lower,
         du.username draft_username, NULL as raw, NULL as cooked, NULL as post_number
       FROM drafts d
+      LEFT JOIN LATERAL json_extract_path_text (d.data::json, 'postId') postId ON TRUE
```

Comment:
oh my ... this is getting more complex, longer term we are going to have to just add some columns to the drafts table

Reply:
Yeah, only way to get to the post's user right now...